doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,162 @@
+              position: "right",
+            },
+          },
+          {
+            element: "#console-rect",
+            popover: {
+              title: "IPython Console",
+              description: 'The <a href="ipythonconsole.html">Console</a> allows you to run your code from the Editor or interactively. You can also use it to control Spyder’s debugger.',
+              position: "left",
+            },
+          },
+          {
+            element: "#help-rect",
+            popover: {
+              title: "Help",
+              description: 'The <a href="help.html">Help</a> pane displays documentation for the objects you are using in the Editor or the IPython Console. To trigger Help, press Ctrl-I (Cmd-I on macOS) with your cursor over an object, or type its name in the object field.',
+              position: "left",
+            },
+          },
+          {
+            element: "#variable-explorer-rect",
+            popover: {
+              title: "Variable Explorer",
+              description: 'The <a href="variableexplorer.html">Variable Explorer</a> allows you to browse and interact with the objects generated running your code. Double-clicking a variable will open a specialized viewer, allowing you to inspect its contents.',
+              position: "left",
+            },
+          },
+          {
+            element: "#plots-rect",
+            popover: {
+              title: "Plots",
+              description: 'The <a href="plots.html">Plots</a> pane shows the figures and images created during your code execution. It allows you to browse, zoom, copy, and save the generated plots.',
+              position: "left",
+            },
+          },
+          {
+            element: "#files-rect",
+            popover: {
+              title: "Files",
+              description: 'The <a href="fileexplorer.html">Files</a> pane lets you browse the contents of the directories on your computer, open them in the Editor, and perform a variety of other file operations.',
+              position: "left",
+            },
+          },
+          {
+            element: "#find-rect",
+            popover: {
+              title: "Find",
+              description: 'The <a href="findinfiles.html">Find</a> pane allows you to view and navigate through all occurrences of your search text in a given directory.',
+              position: "left",
+            },
+          },
+          {
+            element: "#profiler-rect",
+            popover: {
+              title: "Profiler",
+              description: 'The <a href="profiler.html">Profiler</a> helps you optimize your code by determining the run time and number of calls for every function and method used in a file. It allows you to save and compare your results between runs.',
```

Comment:
```suggestion
              description: 'The <a href="profiler.html">Profiler</a> helps you optimize your code by determining the run time and number of calls for every function and method used in a file. It also allows you to save and compare your results between runs.',
```

I think `also` is missing because the last sentence is talking about additional functionality.